config/DS918+/7.0-41890/ramdisk-002-init-script.patch

```diff
@@ -0,0 +1,28 @@
+--- a/linuxrc.syno.impl
```

Comment:
Shouldn't this be a symlink instead? By looking at this it seems that the patch is the same.

Reply:
no thats changes in the files in DSM7.